### ISSUE TITLE
archive multiple NAS destinations

### ISSUE THREAD

**Issue by JonesiKC** [ericaltendorf/plotman#196]
Hi,

I'm currently mv my plots from tmp to a local disk. Then I want to use the Archival function to move it.
But I have multiple NAS server running. How can I configure multiple Archives?
I don't get it :)

Thanks so much for any help!

**Comment by altendky** [comment]
How do you access them?  rsync?  Network mounted drives?

**Comment by JonesiKC** [comment]
two possibilities
mounted via cifs
or
rsyncd running on each NAS
For my point of view the mounted option seems easier for me

Thanks!

**Comment by altendky** [comment]
Yeah, though my personal perception of mounts (based on long ago experience) is that they are generally more likely to flake out or otherwise have issues.  Anyways... rsyncd is what plotman can connect to presently.  We're working on a local rsync-based PR over in #88.  Neither supports multiple destinations, only multiple drives on the destination.  Both are based on being able to ssh to the target and run `df -aBK` to check space availability.

I don't know where the line is between what plotman should grow to support and what chia plot manager already does, but FYI, https://github.com/rjsears/chia_plot_manager is a thing as well.  I haven't looked into it in any detail but it is targeted at high throughput plotting setups.

**Comment by JonesiKC** [comment]
My workaround :) ... I like your way better
It is super fast

```#!/bin/bash
pdir=/media/plots
plist=/tmp/plist.tmp
cd $pdir
find  -name "*.plot" -not -name "*tmp*" -printf '%P\n'  > $plist

cat $plist | while read line
do
        tar cv  $line |pv| ncat  192.168.178.212 7000 && rm $line
done


```
